media-libs/mesa/mesa-9999.ebuild

```diff
@@ -240,7 +241,8 @@ src_unpack() {
 
 pkg_pretend() {
 	if use vulkan; then
-		if ! use video_cards_d3d12 &&
+		if ! use video_cards_asahi &&
```

Comment:
"asahi" should be added to the `ewarn` below. That list becomes comically long though